src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -127,7 +127,7 @@ doAccountInfo(RPC::JsonContext& context)
 
         // The document states that signer_lists is a bool, however
         // assigning any string value works. Do not allow this
-        // This for api v2 onwards
+        // this for api v2 onwards
```

Comment:
:smile: Close.  If you read the sentence, it now reads "... allow this this for ...".  The word "this" appears two times in a row.  Now that neither of the instances of "this" are capitalized feel free to remove either one.

Reply:
oops, I meant to write "this is for api v2 onwards". I just updated the comment for more clarity. Hope the updated one is fine 😄